training/v5p/Llama4-Scout-17B-16E-Maxtext/README.md

```diff
@@ -8,80 +8,59 @@ Please follow this [link](https://github.com/gclouduniverse/reproducibility/tree
+1. Clone [Maxtext](https://github.com/AI-Hypercomputer/maxtext) repo
+```
+git clone https://github.com/AI-Hypercomputer/maxtext.git
+cd maxtext
+git checkout 3eb77db3c94580f56f1b738f8d254b03bd205e35
+```
 
-In step 1, Use the MaxText [tpu-recipes-v0.1.3](https://github.com/AI-Hypercomputer/maxtext/releases/tag/tpu-recipes-v0.1.3) tag to run this recipe:
+2. Run the following commands to build the docker image
 ```
-git checkout tpu-recipes-v0.1.3
+bash docker_build_dependency_image.sh DEVICE=tpu MODE=stable JAX_VERSION=0.7.0
 ```
 
-In step 3, use the jax-stable-stack image containing JAX 0.5.2:
+3. Create your new GCS bucket
+
+This is the GCS folder for storing test results. You can re-use any of your existing GCS buckets. To create a new bucket:
 ```
-BASE_IMAGE=us-docker.pkg.dev/cloud-tpu-images/jax-stable-stack/tpu:jax0.5.2-rev1
-bash docker_build_dependency_image.sh DEVICE=tpu MODE=stable_stack BASEIMAGE=${BASE_IMAGE}
+GCS_PATH=gs://v5p-demo #<your_GCS_folder_for_results>
+gcloud storage buckets create ${GCS_PATH}  --project ${PROJECT}
 ```
 
-## Run workloads
+4. Specify your workload enviroment variables
 
-From the MaxText root directory, start your workload
+```
+export PROJECT=#<your_compute_project>
+export ZONE=#<your_compute_zone>
+export CLUSTER_NAME=#<your_cluster_name>
+export OUTPUT_DIR=gs://v5p-demo/ #<your_GCS_folder_for_results>
+export DEVICE_TYPE=${DEVICE_TYPE} # v5p-256, v5p-512, or v5p-1024
+```
 
+## Run workloads
+
+5. From the MaxText root directory, start your workload:
 ```
 python3 -m benchmarks.benchmark_runner xpk \
     --project=$PROJECT \
     --zone=$ZONE \
-    --device_type=v5p-256 \
+    --device_type=${DEVICE_TYPE} \
     --num_slices=1  \
     --cluster_name=${CLUSTER_NAME} \
     --base_output_directory=${OUTPUT_DIR} \
     --model_name="llama4_scout_dropless_v5p_256" \
     --base_docker_image=maxtext_base_image
 ```
 
-From your workload logs, you should start seeing step time logs like the following:
+6. Check the training log
 
+From your workload logs, you should see step time logs like the following, with the training progresses:
 ```
-completed step: 12, seconds: 31.494, TFLOP/s/device: 251.760, Tokens/s/device: 2080.892, total_weights: 8388608, loss: 10.929
+completed step: 11, seconds: 31.652, TFLOP/s/device: 225.491, Tokens/s/device: 2070.487, total_weights: 33554432, loss: 11.825
 ```
 
-Workload details can be found in `MaxText@tpu-recipes-v0.1.3` [here](https://github.com/AI-Hypercomputer/maxtext/blob/9ca35d7e60b71303b9f6fa885447d32e8a612c47/benchmarks/maxtext_v5p_model_configs.py#L109-L149):
+7. Workload configuration
+
+Workload configuration details can be found [here](https://github.com/AI-Hypercomputer/maxtext/blob/main/benchmarks/maxtext_v5p_model_configs.py) in MaxText GitHub repo. Look for the configuration `llama4_scout_dropless_v5p_256`.
```

Comment:
Add something like: "please note that this configuration is appropriate for v5p-256, v5p-512, and v5p-1024" (if that's the case)